code/game/jobs/job/medical.dm

```diff
@@ -315,3 +315,49 @@
 	satchel = /obj/item/storage/backpack/satchel_med
 	dufflebag = /obj/item/storage/backpack/duffel/medical
 	box = /obj/item/storage/box/engineer
+
+/datum/job/orderly
+	title = "Orderly"
+	flag = JOB_ORDERLY
+	department_flag = JOBCAT_MEDSCI
+	total_positions = 1
+	spawn_positions = 1
+	is_medical = 1
+	supervisors = "the chief medical officer"
+	department_head = list("Chief Medical Officer")
+	selection_color = "#ffeef0"
+	access = list(ACCESS_MEDICAL, ACCESS_MORGUE, ACCESS_SURGERY, ACCESS_CHEMISTRY, ACCESS_VIROLOGY, ACCESS_GENETICS, ACCESS_MINERAL_STOREROOM)
```

Comment:
You said they don't have surgery access but they have it here in their access list. Was that a mistake?

Reply:
No. There are two access lists.
Access is for low pop.
Minimal access is for high pop.

Reply:
I see. All good then.